modules/executive/components/ExecutiveOverviewCard.tsx

```diff
@@ -43,8 +43,13 @@ export default function ExecutiveOverviewCard({
   mkrOnHat
 }: Props): JSX.Element {
   const [voting, setVoting] = useState(false);
+  const [postedDateString, setPostedDateString] = useState('');
   const { comments } = useExecutiveComments(proposal.address);
 
+  useEffect(() => {
+    setPostedDateString(`posted ${formatDateWithoutTime(proposal.date)}`);
+  }, []);
+
```

Comment:
It was throwing a React hydration error because depending on the timezone of the user, the date would defer from the date rendered on the client, that ensures the string is generated only on the client